localstack-core/localstack/openapi.yaml

```
@@ -703,13 +739,17 @@ paths:
                 type: object
           description: Available LocalStack features and AWS services
     head:
+      tags: [internal]
+      operationId: heath
       responses:
         '200':
           content:
             text/plain: {}
           description: ''
     post:
       description: Restart or terminate LocalStack session
+      operationId: restart_or_terminate_session
```

this could be future-proof in case we support more operations

```suggestion
      operationId: manage_session
```